src/views/TreasuryDashboard/components/Graph/Graph.js

```diff
@@ -17,12 +18,12 @@ export const TotalValueDepositedGraph = () => {
       itemType={itemType.dollar}
       itemNames={tooltipItems.tvl}
       dataKey={["totalValueLocked"]}
-      headerText="Total Value Deposited"
+      headerText={t`Total Value Deposited`}
       stopColor={[["#768299", "#98B3E9"]]}
       bulletpointColors={bulletpoints.tvl}
-      infoTooltipMessage={tooltipInfoMessages.tvl}
+      infoTooltipMessage={tooltipInfoMessages().tvl}
       expandedGraphStrokeColor={theme.palette.graphStrokeColor}
-      headerSubText={`${data && formatCurrency(data[0].totalValueLocked)}`}
+      headerSubText={t`${data && formatCurrency(data[0].totalValueLocked)}`}
```

Comment:
translating a formula results in goofy behavior from `lingui` 
![image](https://user-images.githubusercontent.com/80423742/149819421-ee3da2e9-91af-49cb-a772-22b654478a8d.png)

Reply:
it's a number so I don't think it should be translated anyway?